homeassistant/components/sensor/recollect_waste.py

```diff
@@ -0,0 +1,85 @@
+"""
+Support for Recollect Waste curbside collection pickup.
+
+For more details about this platform, please refer to the documentation at
+https://github.com/stealthhacker/python-recollect-waste
+"""
+from datetime import timedelta
+
+import voluptuous as vol
+import homeassistant.helpers.config_validation as cv
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import CONF_NAME
+from homeassistant.helpers.entity import Entity
+from homeassistant.util import Throttle
+
+REQUIREMENTS = ['recollect-waste==1.0.0']
+
+ATTR_PICKUP_TYPES = 'pickup_types'
+ATTR_AREA_NAME = 'area_name'
+CONF_UPDATE_INTERVAL = 'update_interval'
+CONF_PLACE_ID = 'place_id'
+CONF_SERVICE_ID = 'service_id'
+DOMAIN = 'recollect_waste'
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(CONF_PLACE_ID): cv.string,
+    vol.Required(CONF_SERVICE_ID): cv.string,
+    vol.Optional(CONF_NAME): cv.string,
+    vol.Optional(CONF_UPDATE_INTERVAL, default=timedelta(days=1)): (
+        vol.All(cv.time_period, cv.positive_timedelta)),
+})
+
+
+def setup_platform(hass, config, add_entities, discovery_info=None):
+    """Set up the Recollect Waste platform."""
+    add_entities([RecollectWasteSensor(
+        config.get(CONF_NAME),
+        config.get(CONF_PLACE_ID),
```

Comment:
Since this key is required by the schema, access it directly: `config[CONF_PLACE_ID]`